daemon/logger/splunk/splunk.go

```diff
@@ -201,6 +207,9 @@ func ValidateLogOpt(cfg map[string]string) error {
 		case splunkCAPathKey:
 		case splunkCANameKey:
 		case splunkInsecureSkipVerifyKey:
+		case "labels":
```

Comment:
These three new options are common options for all drivers, they don't have const in the code, I just guessed that it will be easier to keep them in that way so it will be easier to search.